package.json

```diff
@@ -5,6 +5,7 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "@extism/runtime-browser": "0.0.1-rc.13",
+    "@tailwindcss/forms": "^0.5.3",
```

Comment:
In my refactor PR i can investigate moving all the dev dependencies out of the production dependencies list.